### ISSUE TITLE
Pagination select have no label element

### ISSUE THREAD

**Issue by Kenzku** [carbon-design-system/carbon#7239]
Similar to this issue: https://github.com/carbon-design-system/carbon/issues/7233

The scanning also reports the `select` tag does not have label, which found on an environment after integrated **pagination** 

Did not see the same tool report this issue on the demo page though: https://react.carbondesignsystem.com/?path=/story/pagination--pagination

but I can see the same html structures in the demo page as in my environment:

This is from the demo page:
<img width="517" alt="Screenshot 2020-11-06 at 13 50 10" src="https://user-images.githubusercontent.com/1387249/98373375-0f7be500-2037-11eb-80e5-faf2c26ef658.png">

Below is from my environment after carbon integration:

<img width="996" alt="Screenshot 2020-11-06 at 13 42 56" src="https://user-images.githubusercontent.com/1387249/98373461-2de1e080-2037-11eb-8b45-551b6787a0f2.png">

<img width="1139" alt="Screenshot 2020-11-06 at 13 44 37" src="https://user-images.githubusercontent.com/1387249/98373397-199de380-2037-11eb-8de9-e26e14b849f3.png">

I do not see a way to add label in the documentation, did I miss something?

**Comment by carmacleod** [comment]
I can't quite tell if your environment still has this label element with `for="id of select element"`:

![image](https://user-images.githubusercontent.com/3331913/98587979-f3777e00-2298-11eb-8762-7c4b80bf30d9.png)

**Comment by Kenzku** [comment]
Hej, yes, there is the `for=id of the select element`

<img width="655" alt="Screenshot 2020-11-09 at 20 14 38" src="https://user-images.githubusercontent.com/1387249/98591537-47e62200-22c8-11eb-8ea1-012a0e90672e.png">

**Comment by emyarod** [comment]
can you create a [reduced test case in Code Sandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox) for easier testing and debugging?

**Comment by tw15egan** [comment]
Closing this out as I am unable to reproduce

<img width="1125" alt="Screen Shot 2021-02-04 at 12 08 31 PM" src="https://user-images.githubusercontent.com/11928039/106949409-c09b2580-66e1-11eb-9b4e-722ce78569c8.png">

If you are still having issues, feel free to reopen with a reduced test case in Code Sandbox so we can take a look. Thanks!
